PyInstaller/building/makespec.py

```diff
@@ -629,9 +629,11 @@ def __add_options(parser):
         "--runtime-tmpdir",
         dest="runtime_tmpdir",
         metavar="PATH",
-        help="Where to extract libraries and support files in `onefile`-mode. If this option is given, the bootloader "
+        help="Where to extract libraries and support files in `onefile` mode. If this option is given, the bootloader "
         "will ignore any temp-folder location defined by the run-time OS. The ``_MEIxxxxxx``-folder will be created "
-        "here. Please use this option only if you know what you are doing.",
+        "here. Please use this option only if you know what you are doing. Note that on POSIX systems, PyInstaller's "
```

Comment:
Hmmm, I'd have thought that the people who *know what they're doing* and the people who need to be told that shellisms don't apply to things that aren't shells are mutally exclusive groups but hey ho, I guess this is what we get for having an option that people think they want but shouldn't use.

On a less cynical note, should the *on POSIX systems* part be there? We don't interpret `$HOME` or `%HOME%` specially on Windows either.

Reply:
The spirit of my request is to provide a simple alternative suggestion to address the limitations for the use case in my previous comment.

Reply:
> On a less cynical note, should the on POSIX systems part be there? We don't interpret `$HOME` or `%HOME%` specially on Windows either.

You'd be surprised:

https://github.com/pyinstaller/pyinstaller/blob/7aea3993f79c9bb6ec1e4faf8cc4ac3ec10292ab/bootloader/src/pyi_utils.c#L239-L251

Reply:
And we could actually do it on POSIX systems as well, via `wordexp()` from standard C library. Although I'd rather not, after seeing that some implementations spawn `/bin/sh` and the like.

Reply:
> You'd be surprised:

And disappointed. But nevermind...